vendor/miniaudio/common.odin

```diff
@@ -191,6 +193,7 @@ format :: enum c.int {
 	s24     = 3,     /* Tightly packed. 3 bytes per sample. */
 	s32     = 4,
 	f32     = 5,
+	count,
```

Comment:
This is not required because `len(format)` would have been equivalent (iff `count` doesn't exist)